bot/cogs/moderation.py

```diff
@@ -1260,6 +1260,11 @@ def _infraction_to_string(self, infraction_object: Dict[str, Union[str, int, boo
         active = infraction_object["active"]
         user_id = infraction_object["user"]
         hidden = infraction_object["hidden"]
+        created = datetime.fromisoformat(infraction_object["inserted_at"].strftime("%Y-%m-%d %H:%M"))
+        if not infraction_object["expires_at"]:
```

Comment:
is this none or false? if none, i think `is None` would be nicer

Reply:
Fair point, fixing now.